parlai/core/worlds.py

```diff
@@ -317,6 +323,12 @@ def get_task_agent(self):
         """
         return self.get_agents()[0]
 
+    def get_model_agent(self):
```

Comment:
this is great, I actually implemented this exact thing in my preprocess branch, will just pull in these changes once merged